airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -447,9 +447,19 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
     quoteString = (quoteString == null ? database.getMetaData().getIdentifierQuoteString() : quoteString);
     database.setSourceConfig(sourceConfig);
     database.setDatabaseConfig(jdbcConfig);
+
+    createDatabase_postHook(database);
     return database;
   }
 
+  public Map<String, String> getConnectionProperties(final JsonNode sourceConfig) {
+    return JdbcDataSourceUtils.getConnectionProperties(sourceConfig);
+  }
+
+  protected void createDatabase_postHook(JdbcDatabase database) throws SQLException {
```

Comment:
Can we just add a quick one line documentation of when/how to call this

Reply:
+1